build-release.xml

```diff
@@ -179,11 +180,11 @@
     </target>
 
     <target name="init-asciidoctor" depends="init-asciidoctor-extensions" unless="skip.doc">
-        <ivy:cachepath pathid="asciidoctor.path" organisation="org.asciidoctor" module="asciidoctor-ant" revision="1.5.4" inline="true"/>
+        <ivy:cachepath pathid="asciidoctor.path" organisation="org.asciidoctor" module="asciidoctor-ant" revision="1.5.3" conf="default" inline="true"/>
         <taskdef uri="antlib:org.asciidoctor.ant" resource="org/asciidoctor/ant/antlib.xml" classpathref="asciidoctor.path">
```

Comment:
Instead of downgrading the version, I think we should try and figure out whatever the issue is with 1.5.4

Reply:
Very well, here's the output from 1.5.4:

```
.../ant-ivy/build-release.xml:202: org.jruby.exceptions.RaiseException: (LoadError) no such file to load -- asciidoctor
	at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:1040)
	at RUBY.require(/Users/gintas/.ivy2/cache/org.asciidoctor/asciidoctor-ant/jars/asciidoctor-ant-1.5.4.jar!/META-INF/jruby.home/lib/ruby/shared/rubygems/core_ext/kernel_require.rb:54)
	at RUBY.(root)(<script>:9)
```